install_openvslam.sh

```diff
@@ -87,7 +90,7 @@ cmake \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_INSTALL_PREFIX=${EXT_DEP}/install/eigen-3.3.4/ \
     ..
-make -j2 --silent
+make -j`nproc` --silent
```

Comment:
This was the default, but prevents a user from being able to use the machine effectively when attempting to build openvslam in the background, hence the limitation to 2.